addons/api/package.json

```diff
@@ -19,7 +19,7 @@
     "format": "yarn run format:js",
     "format:js": "prettier --write '{addon,app,config,stories,tests,mirage}/**/*.js' *.js",
     "start": "ember serve",
-    "test": "ember test",
+    "test": "ember exam --test-port 0 --split=4 --parallel=1 --random",
```

Comment:
@ZedLi when parallel is set to 1, is this equivalent to not being run in parallel?  Should it be 2 or more?  Perhaps I'm misunderstanding?

Reply:
Take a look at this [comment](https://github.com/hashicorp/boundary-ui/pull/1556#discussion_r1063669761), it unfortunately is confusing but no this parallel setting has no affect on the actual number that is running.

Reply:
Is there a reason to leave it when it has no effect?